gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
-#Mon Feb 19 16:01:33 PST 2018
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
-distributionSha256Sum=5c07b3bac2209fbc98fb1fdf6fd831f72429cdf8c503807404eae03d8c8099e5
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```

Comment:
why did you drop the sha check?

Reply:
Forgot to add it back. F-Droid still builds without it as far as I'm aware.

Reply:
That's not about f-droid: The `distributionSha256Sum` check ensures that gradle doesn't execute randomly downloaded code on a local machine. Please add the checksum back.

Reply:
Will do